website/content/docs/commands/auth/enable.mdx

```diff
@@ -57,12 +57,27 @@ flags](/docs/commands) included on all commands.
   configured default lease TTL, or a previously configured value for the auth
   method.
 
+- `-passthrough-request-headers` `(string: "")` - request header values that will
+  be sent to the auth method.
```

Comment:
I see that the audit options have a note about repeating the option, maybe we should put that on these new docs too?

Reply:
Done